README.md

```diff
@@ -39,69 +43,131 @@ Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardwa
 * [MMdnn](https://github.com/Microsoft/MMdnn) : A comprehensive, cross-framework solution to convert, visualize and diagnose deep neural network models. The "MM" in MMdnn stands for model management and "dnn" is an acronym for deep neural network.
 
-## How to deploy
-#### 1 Prerequisites <a name="ref_prerequisites"></a>
-Before start, you need to meet the following requirements:
+## Try OpenPAI
+
+To try OpenPAI quickly, you can follow [Installation](#Installation) below, and skip customization part. You can add more servers as needed after deployed if you keep OpenPAI for using.
+
+It's highly recommended to try OpenPAI on server(s), which has no usage and service. You can deploy OpenPAI on one or multiple server(s), please refer to [here](https://github.com/Microsoft/pai/wiki/Resource-Requirement) for hardware specification.
+
+## Installation
+
+Follow this part to check prerequisites, deploy and validate an OpenPAI cluster.
+
+### Prerequisites
+
+* Ubuntu 16.04 (18.04 should work, but not fully tested.)
+* Assign each server a static IP address, and servers can communicate each other.
+* Server can access internet, especially need to have access to the docker hub registry service or its mirror. Deployment process will pull Docker images of OpenPAI.
+* SSH service is enabled and share the same username/password and have sudo privilege.
+* NTP service is enabled.
+* Recommend not to install docker or docker's version must be higher than 1.26.
+* OpenPAI reserves memory and CPU for service running, so make sure there are enough resource to run machine learning jobs. Check [hardware requirements](https://github.com/Microsoft/pai/wiki/Resource-Requirement) for details.
+* Dedicated servers for OpenPAI. OpenPAI manages all CPU, memory and GPU resources of servers. If there is any other workload, it may cause unknown problem due to insufficient resource.
+
+### Deploy
+
+The default deployment includes minimum steps to deploy an OpenPAI cluster. Base on the default configuration, you can customize OpenPAI deployment for different hardware environments and use scenarios.
+
+#### Deploy with default configurations
+
+If you are deploying a small or medium size cluster, which is less than 20 servers, it's recommended.
+
+If your are deploying a large size cluster, this section is still needed to generate default configuration, then customize on it.
+
+[Deploy with default configurations](docs/pai-management/doc/distributed-deploy.md)
+
+#### Customize deployment
+
+As various hardware environments and different use sencarios, default configuration of OpenPAI may not fit your case. Based on genereted default configration in last section, you can change it before deploying.
+
+[Customize deployment](docs/pai-management/doc/how-to-generate-cluster-config.md#Optional-Step-3.-Customize-configure-OpenPAI)
+
+### Validate deployment
+
+After deployed, it needs to check if key components in health status.
+
```

Comment:
After deployment, it's recommended to run validation to assure the key components are in health status.